libs/ui-toolkit/src/components/explorer-link/explorer-link.tsx

```diff
@@ -0,0 +1,53 @@
+import classNames from 'classnames';
+import type { ReactNode, HTMLProps } from 'react';
+
+const EXPLORER_URL = process.env['NX_EXPLORER_URL'] as string;
```

Comment:
chore (blocking): the eventual idea of this is to be able to publish this library so it should be independent of env config when building. IMO let's invert this dependency with a `base` prop and have the consumer pass in the base, which can be pulled from config.

Reply:
I see. I am not sure if that makes much sense, tbh. In a UI toolkit the general idea should be to stay as agnostic of the data as possible. from the UI's perspective a link is just a string. Considering this, the idea of having an environment variable was also a bit weird, considering the above, but i imagined it's kinda practical in some cases. that's how the `etherscan-link` is working currently ([here](https://github.com/vegaprotocol/frontend-monorepo/blob/master/libs/ui-toolkit/src/components/etherscan-link/etherscan-link.tsx#L4)). It would be nice to aim for some sort of consistency here I think. 

If we're going to create a different toolkit link component for all the possible hosts we want to allow links to point to, and couple it with certain data types, I think it is very counterproductive and also confusing. Assembling the actual href can be easily handled by the host application, it's just a few pieces of strings. But I think it would be nice to have a better understanding of what we're aiming for, maybe we can have a short discussion about this.

Reply:
We will have a follow up to discuss this with the wider team as discussed on slack